### DISCUSSION TITLE
add some style to interaction button

### DISCUSSION THREAD

**Discussion by Heavyrisem** [discordjs/discord.js#10065] [Q&A] [answered]
### Which application or package is this feature request for?

discord.js

### Feature

in discord.js v13
I can send buttons with new line using MessageActionRow, MessageButton
![image](https://github.com/discordjs/discord.js/assets/46806498/361054ab-4c36-49a5-8087-bfa693ba5175)
```ts
new MessageActionRow().addComponents(
                        new MessageButton()
                            .setStyle('PRIMARY')
                            .setLabel(`${i + 1}: ${V.title.slice(0, 50)} (${V.timestamp})`)
                            .setCustomId(JSON.stringify({ videoId: V.videoId, id: interaction.user.id })),
)
```

but discordjs v14 MessageActionRow, MessageButton are seems deprecated
and buttons dosen't create new line between them
![image](https://github.com/discordjs/discord.js/assets/46806498/d0d77c1d-7951-4dc5-b8c5-62826eeb3577)``
```ts
const buttons = [new ButtonBuilder()
        .setCustomId(String(index))
        .setLabel(
          `${title} - ${artists?.at(0)?.name} - ${formatSecondsToTime(
            duration?.totalSeconds ?? -1,
          )}`,
        )
        .setStyle(ButtonStyle.Primary)];
const row = new ActionRowBuilder<ButtonBuilder>().addComponents(buttons);
```

### Ideal solution or implementation

![image](https://github.com/discordjs/discord.js/assets/46806498/361054ab-4c36-49a5-8087-bfa693ba5175)

Can discord.js v14 support this feature?

**Comment by Idris1401** [comment, marked as answer]
you just need to put a button per row, and not all the buttons in the same row

**Comment by VestaHecateDurso** [comment]
const buttons = [new ButtonBuilder()
        .setCustomId(String(index))
        .setLabel(
          `${title} - ${artists?.at(0)?.name} - ${formatSecondsToTime(
            duration?.totalSeconds ?? -1,
          )}`,
        )
        .setStyle(ButtonStyle.Primary)];
const row = new ActionRowBuilder<ButtonBuilder>().addComponents(buttons);

**Comment by VestaHecateDurso** [comment]
const buttons = [new ButtonBuilder()
        .setCustomId(String(index))
        .setLabel(
          `${title} - ${artists?.at(0)?.name} - ${formatSecondsToTime(
            duration?.totalSeconds ?? -1,
          )}`,
        )
        .setStyle(ButtonStyle.Primary)];
const row = new ActionRowBuilder<ButtonBuilder>().addComponents(buttons);
